data/src/main/java/com/architect/coders/mu8/data/characters/CharactersDAO.kt

```diff
@@ -8,10 +8,10 @@ import androidx.room.Query
 @Dao
 interface CharactersDAO {
 
-    @Query("SELECT * FROM CharactersEntity")
+    @Query(value = "SELECT * FROM CharactersEntity")
```

Comment:
you can remove `value =`

Reply:
I actually added it to avoid a warning :P